suite-common/formatters/src/formatters/prepareDateTimeFormatter.ts

```diff
@@ -0,0 +1,15 @@
+import { A, pipe } from '@mobily/ts-belt';
+
+import { makeFormatter } from '../makeFormatter';
+import { FormatterConfig } from '../types';
+import { prepareDateFormatter } from './prepareDateFormatters';
+import { prepareTimeFormatter } from './prepareTimeFormatter';
+
+export const prepareDateTimeFormatter = (config: FormatterConfig) =>
+    makeFormatter<Date | number | null, string | null>(value => {
+        if (!value) return null;
+        const DateFormatter = prepareDateFormatter(config);
+        const TimeFormatter = prepareTimeFormatter(config);
+
+        return pipe([TimeFormatter.format(value), DateFormatter.format(value)], A.join(', '));
```

Comment:
Why it is joined by `,`? How the result look like? Never seen date like `11:00, 27.1.` :D

Reply:
This is just based on the design. Can change it to what you're proposing